include/GraphGenerator.hpp

```diff
@@ -44,27 +42,20 @@ class GraphGenerator
        @param nvertices number of vertices of the graph to be generated
        @param mean_degree average vertex degree of the generated graph
        @param beta probability of rewiring
+       @param seed seed for the random number engine if positive, else random seed
     */
-    GraphGenerator(int nvertices, int mean_degree, double beta);
+    GraphGenerator(int nvertices, int mean_degree, double beta, int seed = -1);
 
     /**
-       @brief Constructor of a random graph generator.
-
-       @param nvertices number of vertices of the graph to be generated
-       @param mean_degree average vertex degree of the generated graph
-       @param beta probability of rewiring
-       @param seed seed for the random number engine
-    */
-    GraphGenerator(int nvertices, int mean_degree, double beta, unsigned int seed);
+        @brief Generate a random graph based on the Watts-Strogatz model
 
+        Generate a ring of nvertices_ vertices, each vertex is connected
+        to the neighboring mean_degree_ vertices (mean_degree/2 on each side)
```

Comment:
I think this description is not correct? This is the initial step, but the final graph is not a ring.